app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: t('books.message.create_success') }
+        format.html { redirect_to book_url(@book), notice: (t 'message.create_success', name: (t 'activerecord.models.book')) }
```

Comment:
```suggestion
        format.html { redirect_to book_url(@book), notice: t('message.create_success', name: t('activerecord.models.book')) }
```

こういう丸かっこの使い方の方が主流かもしれません

Reply:
`(`の使い方を変更しました。
```ruby
format.html { redirect_to book_url(@book), notice: t('message.create_success', name: t('activerecord.models.book')) }
```